### PR TITLE
[tempo-distributed] switch to new overrides format

### PR DESCRIPTION
**Pull request by farodin91**

### CODE REVIEW

**Comment by AlexDCraig** [comment]
@farodin91 @mapno @swartz-k @BitProcessor @faustodavid Can we get traction on this PR? You can't upgrade right now if you're using global overrides without some wonky workaround

**Review by mapno — APPROVED** [review]
Thanks for the upgrade.

**Comment by mapno** [comment]
If we can get another approval I'll merge 🙏

**Comment by AlexDCraig** [comment]
@swartz-k @BitProcessor @faustodavid can we get another approver for this one?

**Comment by faustodavid** [comment]
Thanks for the contribution, @farodin91. Can you please update the branch and resolve the conflicts?

File: charts/tempo-distributed/Chart.yaml
Comment (YayBurritos) [inline charts/tempo-distributed/Chart.yaml:5]: Looks like the latest tempo-distributed chart version is `1.8.4` (https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/Chart.yaml), so I guess this one will become `1.8.5`?

**Review by faustodavid — APPROVED** [review]
thanks @farodin91 LGTM

**Pull request merged**
